packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionContainerStartCommand.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Uri } from 'vscode';
+import {
+  FilePathGatherer,
+  SfdxCommandlet,
+  SfdxWorkspaceChecker
+} from '../util';
+import {
+  ForceFunctionContainerStartExecutor,
+  validateStartFunctionsUri
+} from './forceFunctionStart';
+
+export const CONTAINER_START_TEXT_KEY = 'force_function_container_start_text';
+export const FUNCTION_CONTAINER_LOG_NAME = 'force_function_container_start';
+
+/**
+ * Executes sfdx run:function:start --verbose
+ * @param sourceUri
+ */
+export async function forceFunctionContainerStartCommand(sourceUri?: Uri) {
```

Comment:
Did some refactoring here to try and separate some concerns.  Previously we have one file that exported both the command and executor. I moved those to different files to enable easier unit testing of each piece of the puzzle.